Source/BusinessMomentsAdaptorImage/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "businessmomentsadaptorimage",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "NODE_ENV=test mocha --check-leaks -r tsconfig-paths/register -r ts-node/register \"**/for_*/**/*.ts\"",
```

Comment:
For an unknown reason to me, I need to pass the NODE_ENV variable and check if the code is in the test scenario or not. It seems during the test, the code is trying to listen to the same port twice and I will get a "port already in use" error. While I expect it to run only once and the error has no sense to me. I have solved it in this way, by making the server.ts not listen to the port while testing. Think there should be better ways to solve the issue.
Any comments?